vshn/website
# feat: integrate recruitee jobs widget

**Describe what changes this pull request brings**
This pull request brings the Recruitee jobs widget integration and widget customization, replacing the Freshteam widget.


**Describe any additional changes**
N/A


**Steps to test**
1. Open [the page](https://websitemaster20486-recruiteeintegration.gtsb.io/jobs/)
2. Confirm that everything looks and works as expected


**Screenshots**


**Checklist**



**TODO**
- [x] add company id to jobs widget


**References**
N/A
